smart-contracts/contracts/PublicLock.sol

```diff
@@ -219,12 +222,7 @@ contract PublicLock is ILockCore, ERC165, IERC721, IERC721Receiver, Ownable {
     if (previousExpiration == 0) {
       // The recipient did not have a key previously
       owners.push(_recipient);
-      // Note: not using the tokenId above since ATM we do not transfer tokenId's
-      // this will change when we decouple tokenId from address
-      ownerByTokenId[uint(_recipient)] = _recipient;
-      // At the moment tokenId is the user's address, but as we work towards ERC721
-      // support this will change to a sequenceId assigned at purchase.
-      keyByOwner[_recipient].tokenId = uint(_recipient);
```

Comment:
We do still need to set the tokenId in the struct here.

Reply:
Good point, I think we are missing a test case here.  To test, we could add to getTokenIdFor.js - just confirm that the result for the recipient of a transferFrom is not 0.